pkg/diagnose/firewall.go

```diff
@@ -288,3 +304,32 @@ func getTargetPort(submariner *v1alpha1.Submariner, endpoint *subv1.Endpoint, tg
 		return 0, fmt.Errorf("could not determine the target port for cable driver %q", endpoint.Spec.Backend)
 	}
 }
+
+func getLbNodePort(clusterInfo *cluster.Info, endpoint *subv1.Endpoint, tgtport TargetPort, status reporter.Interface) (int32, bool) {
+	var lbNodePort int32
```

Comment:
This is never set and is only used as an uninitialized return value so we should just remove it and return 0.